bin/pycbc_single_template

```diff
@@ -85,19 +85,22 @@ def select_segments(fname, anal_name, data_name, ifo, time, pad_data):
 
 parser = argparse.ArgumentParser(usage='',
     description="Single template gravitational-wave followup")
-parser.add_argument('--version', action=pycbc.version.Version) 
+parser.add_argument('--version', action=pycbc.version.Version)
 parser.add_argument('--output-file', required=True)
 parser.add_argument('--subtract-template', action='store_true')
-parser.add_argument("-V", "--verbose", action="store_true", 
+parser.add_argument("-V", "--verbose", action="store_true",
                   help="print extra debugging information", default=False )
 parser.add_argument("--low-frequency-cutoff", type=float,
                   help="The low frequency cutoff to use for filtering (Hz)")
 parser.add_argument("--chisq-bins", default="0", type=str, help=
                     "Number of frequency bins to use for power chisq.")
 parser.add_argument("--minimum-chisq-bins", default=0, type=int, help=
                     "If the chisq bin formula fails, default to this number.")
-parser.add_argument("--trigger-time", type=float, default=0,
-                  help="The central time of the trigger to use.")
+# flag for trigger time not being entered
+stupid_time = -999999999.
+parser.add_argument("--trigger-time", type=float, default=stupid_time,
```

Comment:
Why do you need stupid_time? Could you not use:
```
default=None
```
and then check for:
```
if opt.trigger_time is [not] None
```
below

Reply:
I suppose so.

Reply:
See the revised diff.